src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/NodeBase.java

```diff
@@ -0,0 +1,86 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import java.util.LinkedList;
+
+public class NodeBase  {
```

Comment:
Raz - for the future, configuration for nodes might be better in a configuration file

Reply:
I think it's probably worth to change it now

Reply:
I also think we should add some visual "map" that will help us keep track on those